dockerbuild/Dockerfile-backend

```diff
@@ -63,8 +64,32 @@ WORKDIR /modeldb/server
 RUN ../scripts/gen_thrift_file.sh java '../thrift/ModelDB.thrift' './src/main/thrift/' && \
     mvn clean compile -Dthrift_version=0.10.0
 
-# This would be better as an ENTRYPOINT with 'mongo' the CMD param, but
-# `exec mvn exec:java ...` leads to a classpath issue that isn't immediately
+
+RUN mvn package -Dthrift_version=${THRIFT_VERSION} -Dmaven.test.skip=true
+
+FROM openjdk:8
+
+WORKDIR /modedlb/server
+
+EXPOSE 6543
+
+COPY --from=builder /modeldb/server/target/modeldb-*.jar /modeldb/modeldb.jar
+COPY --from=builder /modeldb/server/src/main/resources/reference-docker.conf /modeldb/reference.conf
```

Comment:
Per my original comment: You should be able to copy from the docker staging area (`COPY server/src/main/resources/reference-docker.conf /modeldb/reference.conf`).

Reply:
Yes! For some reason copying without the --from=builder didn't work when I tried the first time around, but it works well now!